constants/battle_tower_constants.asm

```diff
@@ -43,8 +43,8 @@ DEF BATTLE_TOWER_STRUCT_LENGTH EQU NAME_LENGTH + BATTLETOWER_PARTY_LENGTH * NICK
 	const BATTLETOWERACTION_SAVEOPTIONS
 
 ; BattleTowerAction return values
-; - from BATTLETOWERACTION_CHECKMOBILEEVENT
-DEF MOBILE_EVENT_OBJECT_GS_BALL EQU $b
+; - from BATTLETOWERACTION_GSBALL
+DEF GS_BALL_FLAG EQU $b
```

Comment:
no, no, the GS ball flag holds the value "gs ball available", that part was fine.